tests/accumulate-chain.rs

```diff
@@ -0,0 +1,54 @@
+mod common;
+
+use expect_test::expect;
+use salsa::{Accumulator, Database};
+use test_log::test;
+
+#[salsa::accumulator]
+struct Log(#[allow(dead_code)] String);
+
+#[salsa::tracked]
+fn push_logs(db: &dyn Database) {
+    push_a_logs(db);
+}
+
+#[salsa::tracked]
+fn push_a_logs(db: &dyn Database) {
+    Log("log a".to_string()).accumulate(db);
```

Comment:
Not sure if it matters, but IIRC the idiomatic syntax is `Log::push(...)`.

Reply:
I followed the pattern in https://github.com/salsa-rs/salsa/blob/master/tests/accumulate-dag.rs

Not sure if it's worth adding a `push` method to this local `Log` struct as it would only be used twice, but I'm happy to do it if you prefer.

Reply:
No, the "idiomatic syntax" changed to use the `accumulate` method. I think I removed `push` altogether.